nwb_web_gui/dashapps/pages/converter/converter.py

```diff
@@ -1,12 +1,13 @@
+import json
+from pathlib import Path
+
 import dash
+import dash_bootstrap_components as dbc
 import dash_html_components as html
-import dash_core_components as dcc
 from dash.dependencies import Input, Output, State, ALL, MATCH
-import dash_bootstrap_components as dbc
-import json
+
+from ....dashapps.utils.make_components import make_modal
```

Comment:
use relative imports where possible. The exception to this is when building tests use absolute imports